scripts/build-appimage.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh -ex
```

Comment:
I thought that generating AppImage  apps it was much more difficult since the detailed specification. That is nice to see.